pyrdp/enum/rdp.py

```diff
@@ -347,6 +347,28 @@ class NegotiationProtocols(IntFlag):
     EARLY_USER_AUTHORIZATION_RESULT = 0b00001000
 
 
+class NegotiationFailureCode(IntEnum):
```

Comment:
I like when we write the source for those flags in the class' documentation. In this case its https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/1b3920e7-0116-4345-bc45-f2c4ad012761